src/tcfv2/stub.js

```diff
@@ -1,297 +1,91 @@
 /* eslint-disable -- this is third party code */
 /* istanbul ignore file */
 
-// https://documentation.sourcepoint.com/web-implementation/web-implementation/multi-campaign-web-implementation#stub-file
+// https://documentation.sourcepoint.com/web-implementation/web-implementation/sourcepoint-gdpr-and-tcf-v2-support/gdpr-and-tcf-v2-setup-and-configuration_v1.1.3
 
 export const stub = () => {
-	//GDPR TCF v2 Stub file
-	!(function (t) {
-		var e = {};
-		function n(r) {
-			if (e[r]) return e[r].exports;
-			var o = (e[r] = { i: r, l: !1, exports: {} });
-			return t[r].call(o.exports, o, o.exports, n), (o.l = !0), o.exports;
+	var e = function () {
```

Comment:
Could we disable auto-formatting here? 

```suggestion
	// prettier-ignore
	var e = function () {
```

Reply:
Would we disable auto-formatting in order to include the minified stub on a single line?